### ISSUE TITLE
class or module for F# NUnit template

### ISSUE THREAD

**Issue by zanaptak** [nunit/dotnet-new-nunit#27]
Ping @cartermp and @KevinRansom since you contributed/commented here before, is there any reason to keep class boilerplate by default here? It could just be a module (like xunit template):

```fs
module Company.TestProject1

open NUnit.Framework

[<SetUp>]
let Setup () =
    ()

[<Test>]
let Test1 () =
    Assert.Pass()
```

vs. current:
```fs
namespace Company.TestProject1

open NUnit.Framework

type TestClass () =

    [<SetUp>]
    member this.Setup () =
        ()

    [<Test>]
    member this.Test1 () =
        Assert.Pass()
```

**Comment by cartermp** [comment]
No real reason. I think it'd be better to have them be module-bound function. That's what we use in the F# test suite: https://github.com/dotnet/fsharp/blob/master/tests/fsharp/Compiler/Language/AnonRecordTests.fs

**Comment by halex2005** [comment]
Since #27 is merged, changes released under v1.6.3. I'll close this issue. Feel free to reopen the issue if something is wrong.
